runner/src/runtime/local-build.ts

```diff
@@ -177,12 +186,15 @@ export const tsToExports = async (
 
   let localImports: Record<string, any> | undefined;
   try {
-    localImports = await ensureRequires(js);
+    localImports = await ensureRequires(js, config);
   } catch (e) {
     const error = e as Error;
+    if (!error.stack) {
+      throw new Error("No stack trace.");
+    }
     // Add source location context if possible
     const errorWithContext = mapSourceMapsOnStacktrace(error.stack);
```

Comment:
this was my code, but i think this is wrong, as `mapSourceMapsOnStacktrace` returns a string. it should probably be

```ts
if (error.stack)
  throw { ...error, stack:  mapSourceMapsOnStacktrace(error.stack) }
else
  throw error;
```

Reply:
Fixed